templates/partners/partial/_co-sell-with-canonical.html

```diff
@@ -0,0 +1,15 @@
+<section class="p-section">
```

Comment:
Looking over the designs for the rest of the pages and I'm not sure if this should be a partial necessarily. As a pattern it's not repeated nearly as much as the logos or breadcrumbs

Reply:
I found it in 3 places!

One of them was a bit different like it wasn't updated with the other two so I thought this would be a neat way to keep everything in sync!